config.py.example

```diff
@@ -13,6 +13,10 @@
 # limitations under the License.
 #
 # Configuration File
+device_config = {
+  'led_pin': 1
```

Comment:
Should this be 5?

Reply:
With a Huzzah32 is 1. 5 with which board?

Reply:
Using the pin for the Huzzah32 makes sense, I'm using a Lolin Wemos32